scripts/fixxxers/operator/10-deps-go-mod-tidy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+
+source scripts/lib.sh
+
+find operator/ -type f -name go.mod -printf "%h\n" | while read -r dir
```

Comment:
```suggestion
find . -type f -name go.mod -printf "%h\n" | while read -r dir
```

Reply:
I'll do something line this separately, in a different file.